sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/geojson/GeoLinearRing.java

```diff
@@ -47,10 +49,15 @@ public GeoLinearRing(List<GeoPosition> coordinates) {
      * @return An unmodifiable representation of the {@link GeoPosition geometric positions} representing this linear
      * ring.
      */
-    public GeoArray<GeoPosition> getCoordinates() {
+    public List<GeoPosition> getCoordinates() {
         return coordinates;
     }
 
+//    For now, getting coordinates will return built-in types. This is being left as it may be revived.
+//    GeoArray<GeoPosition> getCoordinates() {
+//        return coordinates;
+//    }
```

Comment:
Since the method is package-private anyway, we can uncomment and keep it or just delete and add it back when needed.